base_module_record/i18n/ar.po

```diff
@@ -0,0 +1,267 @@
+# Translation of OpenERP Server.
```

Comment:
can you regenerate ALL the PO files please?

Reply:
Done